### DISCUSSION TITLE
whether can use window in zustand store?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [pmndrs/zustand#1926] [Q&A] [answered]
whether can use window in zustand store?

when I use `window` in zustand store file:
```
...
const ipcRenderer = (window as unknown as Window).ipcRenderer  // there get error

```
![image](https://github.com/pmndrs/zustand/assets/12727582/a43435c9-512f-4899-91d2-5cdc2f49203c)

I get error below: 

```
Server Error
ReferenceError: window is not defined
```

![image](https://github.com/pmndrs/zustand/assets/12727582/df8cd2d1-5385-43f7-98e3-556797640ca2)

**Comment by dbritto-dev** [comment, marked as answer]
@liaodalin19903 should be just `ipcRenderer`
